foreman/foreman.spec

```diff
@@ -372,6 +374,8 @@ Requires: %{?scl_prefix_ror}rubygem(uglifier) >= 1.0.3
 Requires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 Requires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 Requires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+Requires: %{?scl_prefix_ror}rubygem(sprockets) >= 3
```

Comment:
http://projects.theforeman.org/issues/13074 ought to be fixed as this should probably be in the main foreman deps, not assets only.